src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx

```diff
@@ -346,8 +346,8 @@ export const ReviewCollapsible = ({
 
   const getMultiSelectionReviewContent = () => {
     const multiSelectionReviewLevel = baseReviewLevel as MultiSelectionReviewLevel;
-    const joinedNewParts = [];
-    const joinedOldParts = [];
+    const joinedNewParts: string[] = [];
+    const joinedOldParts: (string | undefined)[] = [];
```

Comment:
Could we just not push the undefined values? If we have an array with only `undefined`, we could end up with a value after joining like `', '`